lms/static/sass/course-list.scss

```diff
@@ -6,20 +6,22 @@
  */
 
 :root {
+  
+  //new-layout
+  --primary-color:#1868B9;
+  --background-2-color:#EBF3FA;
+  --line-color:#DEE2E6;
```

Comment:
tu też nazwa do zmiany, jakiego elementu to linia, bo nie jest to linia globalna w użyciu